daemon/nvidia_linux.go

```diff
@@ -17,7 +17,10 @@ import (
 
 var errConflictCountDeviceIDs = errors.New("cannot set both Count and DeviceIDs on device request")
 
-const nvidiaHook = "nvidia-container-runtime-hook"
+const (
+	nvidiaHook                        = "nvidia-container-runtime-hook"
+	amdContainerRuntimeExecutableName = "amd-container-runtime"
```

Comment:
If we split (see other comment); probably cleaner to have this const defined in the `amd_linux` file.

Possibly rename both files to have a common prefix (`gpu_amd_linux.go`, `gpu_nvidia_linux.go`) for easier discoverability that they go together.

Reply:
Yes, that's fine.